tuf/keys.py

```diff
@@ -1232,8 +1266,8 @@ def encrypt_key(key_object, password):
  
   # check_crypto_libraries() should have fully verified _GENERAL_CRYPTO_LIBRARY.
   else: # pragma: no cover
-    message = 'Invalid crypto library: ' + repr(_GENERAL_CRYPTO_LIBRARY) + '.'
-    raise tuf.UnsupportedLibraryError(message) 
+    raise tuf.UnsupportedLibraryError('Invalid crypto library:'
+      ' ' + repr(_GENERAL_CRYPTO_LIBRARY) + '.') 
```

Comment:
Unnecessary explicit concatenation, unnecessary string separation - multiple concatenation formats.

Should be this:
 `raise tuf.UnsupportedLibraryError('Invalid crypto library: ' repr(_GENERAL_CRYPTO_LIBRARY) '.')`

instead of the current:
 `raise tuf.UnsupportedLibraryError('Invalid crypto library:'  ' ' + repr(_GENERAL_CRYPTO_LIBRARY) + '.')`